### PR TITLE
Extrinsic unvest call and vesting details

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

> Unvesting with metamask and vesting details dialog

image
image

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Adds**
- extrinsic unvest call
- vesting details dialog

**Fixes**
- vested amount on balance page

### CODE REVIEW

**Review by hoonsubin — CHANGES_REQUESTED** [review]
Some style issues.

File: src/components/balance/PlmBalance.vue [hunk 1]
```diff
@@ -182,21 +160,35 @@
         </div>
       </div>
     </div>
+    <ModalVestingInfo
```
[the remark this answers is not in the document]
Comment (bobo-k2) [reply src/components/balance/PlmBalance.vue:163]: Dialog should display all vesting schedules. Let me check

[the remark this answers is not in the document]
Comment (hoonsubin) [reply src/components/balance/PlmBalance.vue:163]: Screen Shot 2022-01-20 at 3 26 20 PM
Oh, my bad, I can confirm that it does. But what about the list length? I was expecting a constant sized list box

File: src/components/balance/PlmBalance.vue [hunk 2]
```diff
@@ -85,46 +68,53 @@
...
+          </div>
+          <div>
+            <p class="tw-font-bold tw-text-right">
+              <span class="tw-text-2xl md:tw-text-xl xl:tw-text-2xl tw-leading-tight">
+                <format-balance :balance="accountData?.vested" />
+              </span>
+            </p>
+          </div>
+        </div>
+        <Button :small="true" class="tw-self-end tw-mt-1" @click="showVestingInfo">
```
Comment (hoonsubin) [inline src/components/balance/PlmBalance.vue:115]: Screen Shot 2022-01-20 at 4 18 55 PM

I like the staked view of the numbers. But the button position does not look very good. Can we move the `Info` button somewhere else? For example, next to the `Claimable` next or next to the faucet button and change the name of the button.

**Review by hoonsubin — CHANGES_REQUESTED** [review]

**Review by hoonsubin — APPROVED** [review]
LGTM
Great work! There are some minor UX improvements we can make, but we can do that later when we redesign the portal.

**Pull request merged**